content/docs/command-reference/add.md

```diff
@@ -344,3 +342,44 @@ $ tree .dvc/cache
 
 Only the hash values of the `dir/` directory (with `.dir` file extension) and
 `file2` have been cached.
+
+# Example: Transfer to remote storage
+
+When you want to include a remote location (like some file in a s3 bucket, or a
+Google Drive directory) to your project, but don't want DVC to control the given
```

Comment:
`but don't want DVC to control the given` it's a bit confusing to me- we do want to control this data, may be not location itself indeed.

@isidentical you had some good explanation in the `Transferring data directly to remote storage` - I would just move it here if we got with an example here. It will make both `import-url` and `add` consistent:

- long explanations go into examples
- short notice with a link where it is appropriate in the Description

Reply:
I've moved the section as is from import-url, and just polished it for add. Though not sure `let your skip download` part makes sense now or not since that wasn't the previous behavior

Reply:
Good idea copying from `import-url` which we already reviewed and just adapt for add. I thin that "skip download" makes sense but we should just emphasize that we mean skip downloading (or copying) **to the local cache**. I'll leave some copy edits separately...